stories/documentation/navigation/table-of-content/table-of-content-basic.stories.ts

```diff
@@ -1,45 +1,27 @@
 import { Meta, Story } from '@storybook/angular';
 
 interface TableOfContentBasicStory {
-	mod: string;
 }
 
 export default {
 	title: 'Documentation/Navigation/TableOfContent/Basic',
-	argTypes: {
-		mod: {
-			options: ['', 'mod-grey'],
-			control: {
-				type: 'radio',
-			}
-		},
-	},
 } as Meta;
 
 function getTemplate(args: TableOfContentBasicStory): string {
-	const classes = [args.mod].filter(Boolean).join(' ');
 	return `
-	<nav class="tableOfContent ${classes}">
-		<a class="tableOfContent-item is-active">Section 1</a>
-		<a class="tableOfContent-item">Section 2</a>
-		<a class="tableOfContent-item">Section 3</a>
-		<a class="tableOfContent-item">Section 4</a>
+	<nav class="tableOfContent">
+		<a href="#" class="tableOfContent-item is-active">Section 1</a>
```

Comment:
I don't see the href added [here](http://lucca-front.lucca.local/PR-1613/?path=/story/qa-table-of-content--basic), why?

Reply:
Just an oversight, fixed.